src/main/groovy/org/javafxports/jfxmobile/plugin/JFXMobilePlugin.groovy

```diff
@@ -370,7 +369,7 @@ class JFXMobilePlugin implements Plugin<Project> {
 
                 // configure ios boot classpath
                 project.tasks.compileIosJava {
-                    options.bootstrapClasspath = project.configurations.iosBootclasspath
+                    dependsOn project.tasks.compileJava
```

Comment:
You can completely remove this section. The dependency is implicitly set because the classpath of the ios source set is extended with the output of the main source set (see https://github.com/javafxports/javafxmobile-plugin/blob/master/src/main/groovy/org/javafxports/jfxmobile/plugin/JFXMobilePlugin.groovy#L140)

The reason it couldn't find the JDK classes is because an empty bootstrap classpath was set, which overrides the default bootstrap classpath.

Reply:
Ok, it makes sense.